.gitlab-ci.yml

```diff
@@ -688,8 +688,7 @@ zombienet-tests-parachains-pvf:
     - /home/nonroot/zombie-net/scripts/run-test-env-manager.sh
         --github-remote-dir="${GH_DIR}"
         --test="0001-parachains-pvf.feature"
-  allow_failure:                   true
-  retry: 2
+  allow_failure:                   false
```

Comment:
Can be removed, the default value is `false`

Reply:
I prefer explicit to implicit, that way, if there is a regression, it avoids the detour via the gitlab y'all spec docs